packages/davinci-client/package.json

```diff
@@ -8,5 +8,12 @@
   "dependencies": {
     "@forgerock/javascript-sdk": "*",
     "@reduxjs/toolkit": "^2.2.5"
+  },
+  "exports": {
+    ".": {
+      "import": "./src/index.ts",
```

Comment:
I think these should be js files and relative to the dist directory, so if this package.json is not in the `dist` directory we would either want to a) copy the package.json file into the dist directory or b) point the package.json exports to the dist/src/index.js file

Reply:
unless ofc, we need to edit these after the build process, i believe i do do that or used to do that elsewhere